dev-util/lxqt-build-tools/files/lxqt-build-tools-0.10.0-glib-2.72.patch

```diff
@@ -0,0 +1,25 @@
+From 62b77bb807aa2ac2c1df8eec372df771da8d2203 Mon Sep 17 00:00:00 2001
```

Comment:
Please include link to bug & commit at the top of the patch.

Reply:
eh, I see the FILESDIR comment, good enough to get it in

Reply:
I had that in mind, too, but then I forgot, and it's all a hue.

Thanks!